zerver/lib/webhooks/git.py

```diff
@@ -1,24 +1,26 @@
 from typing import Optional, Any, Dict, List, Text
+from collections import defaultdict
 
 SUBJECT_WITH_BRANCH_TEMPLATE = u'{repo} / {branch}'
 SUBJECT_WITH_PR_OR_ISSUE_INFO_TEMPLATE = u'{repo} / {type} #{id} {title}'
 
 EMPTY_SHA = '0000000000000000000000000000000000000000'
 
-COMMITS_LIMIT = 10
+COMMITS_LIMIT = 20
 COMMIT_ROW_TEMPLATE = u'* [{commit_short_sha}]({commit_url}): {commit_msg}\n'
 COMMITS_MORE_THAN_LIMIT_TEMPLATE = u"[and {commits_number} more commit(s)]"
 
-PUSH_PUSHED_TEXT_WITH_URL = u"[pushed]({compare_url})"
-PUSH_PUSHED_TEXT_WITHOUT_URL = u"pushed"
-PUSH_COMMITS_MESSAGE_TEMPLATE = u"""{user_name} {pushed_text} to branch {branch_name}
-
-{commits_data}
-"""
+PUSH_PUSHED_TEXT = u"pushed {no_of_commits} commits"
+PUSH_COMMITS_MESSAGE_TEMPLATE = u"{user_name} {pushed_text} to branch {branch_name}."
+COMMITS_DATA = u"{commits_data}"
+COMPARE_URL = u"{compare_url}"
+OTHER_COMMITTERS_MESSAGE_TEMPLATE = u"Commits by{others}"
```

Comment:
when i am creating the variable `others` in `get_all_committers` function, i have included a space at the start